### PR TITLE
Search result arrow

### PR DESCRIPTION
**Pull request by lifez**
Changes proposed in this pull request:
- Add support arrow key in `Search Results` page
- Add function `isSearchResultPage`
Connected to #14

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -115,7 +118,12 @@
             const isDown = ev.keyCode === 40;
 
             // Get all items.
-            const itemList = document.getElementsByClassName('itemList')[0];
+            var itemList
```
Comment (martellaj) [inline script.js]: `let` instead of `var`, please.

Comment (martellaj) [reply script.js]: And don't forget the ";".

File: script.js [hunk 2]
```diff
@@ -115,7 +118,12 @@
...
+            if (isSearchResultPage()){
```
Comment (martellaj) [inline script.js]: nit: Space before brace.

File: script.js [hunk 3]
```diff
@@ -115,7 +118,12 @@
...
+                itemList = document.getElementsByClassName('paginated-item-list')[0];
+            }else {
```
Comment (martellaj) [inline script.js]: nit: Space after brace.

File: script.js [hunk 4]
```diff
@@ -129,6 +137,7 @@
                 tapElement(div);
             }
         } catch (error) {
+            log(error)
```
Comment (martellaj) [inline script.js]: nit: Semi-colon.

File: script.js [hunk 5]
```diff
@@ -190,6 +199,27 @@
     }
 
     /**
+     * Search for the current item to see what other ones on the market are going for.
+     */
+    function buyNow() {
```
Comment (martellaj) [inline script.js]: Looks like this is from your other PR. Please remove these changes.

**Comment by lifez** [comment]
Done!

**Pull request merged**